gulpfile.js

```diff
@@ -46,9 +48,7 @@ gulp.task('build:backend:script', () => {
 
 gulp.task('build:backend:style', () => {
 	return gulp.src(['./packages/backend/src/server/web/style.css', './packages/backend/src/server/web/bios.css', './packages/backend/src/server/web/cli.css'])
-		.pipe(cssnano({
-			zindex: false
-		}))
+		.pipe(postcss([ autoprefixer(), cssnano({zindex: false}) ]))
```

Comment:
```suggestion
		.pipe(postcss([cssnano({ zindex: false })]))
```

convensional commits のフォーマットで type が refactor とすると `autoprefixer` の追加はスコープを外れると思います
（それはそれとして autoprefixer の導入可否については合意が未着地 https://github.com/misskey-dev/misskey/pull/7530#issuecomment-846376872 なので CSS ポストプロセッシング周りの相談機会は欲しいです）